could be upstreamed, but upstream seems dead
with lists.windowmaker.org down and the last git commit is from March

commit e9ab99877b71669a75c1c1a43c1fb24cb32c1bdf
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Tue Dec 19 21:13:21 2017 +0100

    texi2txt: Use ChangeLog date instead of build date

    in order to make builds reproducible.
    See https://reproducible-builds.org/ for why this is good.

    This date call works with GNU date and BSD date.

    Without this patch, /usr/share/doc/packages/WindowMaker/README.i18n
    will differ in the line
      Published by The Window Maker team on ...

    ---
    An alternative solution could use the $SOURCE_DATE_EPOCH variable
    defined in https://reproducible-builds.org/specs/source-date-epoch/

Index: autogen.sh
===================================================================
--- autogen.sh.orig
+++ autogen.sh
@@ -1,8 +1,10 @@
 #!/bin/sh
 
 # Generate the documentation about compiling Window Maker
-./script/generate-txt-from-texi.sh "doc/build/Compilation.texi"  -o "INSTALL-WMAKER"
-./script/generate-txt-from-texi.sh "doc/build/Translations.texi" -o "README.i18n"
+### Use date of INSTALL file instead of build (current/compile) date
+InstallDate="`LANG=C date -u -r ./INSTALL '+%d %B %Y' | sed -e 's,^0,,' `"
+./script/generate-txt-from-texi.sh -d "$InstallDate" "doc/build/Compilation.texi"  -o "INSTALL-WMAKER"
+./script/generate-txt-from-texi.sh -d "$InstallDate" "doc/build/Translations.texi" -o "README.i18n"
 
 # Change date of the files to the past so they will be regenerated by 'make'
 touch -d '2000-01-01' INSTALL-WMAKER README.i18n
